crates/storage/src/state.rs

```diff
@@ -1,3011 +1,1316 @@
+    //                     StarknetBlocksTable::get_latest_hash_and_number(tx).unwrap(),
+    //                     Some((latest.hash, latest.number))
+    //                 );
+    //             });
+    //         }
+
+    //         #[test]
+    //         fn none() {
+    //             let storage = Storage::in_memory().unwrap();
+    //             let mut connection = storage.connection().unwrap();
+    //             let tx = connection.transaction().unwrap();
+
+    //             assert_eq!(
+    //                 StarknetBlocksTable::get_latest_hash_and_number(&tx).unwrap(),
+    //                 None
+    //             );
+    //         }
+    //     }
+
+    //     mod get_hash {
+    //         use super::*;
+
+    //         mod by_number {
+    //             use super::*;
+
+    //             #[test]
+    //             fn some() {
+    //                 with_default_blocks(|tx, blocks| {
+    //                     for block in blocks {
+    //                         let result =
+    //                             StarknetBlocksTable::get_hash(tx, block.block.number.into())
+    //                                 .unwrap()
+    //                                 .unwrap();
+
+    //                         assert_eq!(result, block.block.hash);
+    //                     }
+    //                 })
+    //             }
+
+    //             #[test]
+    //             fn none() {
+    //                 with_default_blocks(|tx, blocks| {
+    //                     let non_existent = blocks.last().unwrap().block.number + 1;
+    //                     assert_eq!(
+    //                         StarknetBlocksTable::get(tx, non_existent.into()).unwrap(),
+    //                         None
+    //                     );
+    //                 });
+    //             }
+    //         }
+
+    //         mod latest {
+    //             use super::*;
+
+    //             #[test]
+    //             fn some() {
+    //                 with_default_blocks(|tx, blocks| {
+    //                     let expected = blocks.last().unwrap().block.hash;
+
+    //                     let latest =
+    //                         StarknetBlocksTable::get_hash(tx, StarknetBlocksNumberOrLatest::Latest)
+    //                             .unwrap()
+    //                             .unwrap();
+    //                     assert_eq!(latest, expected);
+    //                 })
+    //             }
+
+    //             #[test]
+    //             fn none() {
+    //                 let storage = Storage::in_memory().unwrap();
+    //                 let mut connection = storage.connection().unwrap();
+    //                 let tx = connection.transaction().unwrap();
+
+    //                 let latest = StarknetBlocksTable::get(&tx, BlockId::Latest).unwrap();
+    //                 assert_eq!(latest, None);
+    //             }
+    //         }
+    //     }
+    // }
+
+    // mod starknet_events {
```

Comment:
Where do these tests go?

I think we have some tests for other storage functionality but didn't find any for events...

Reply:
I'll be moving the tests across still; sorry really should have done that before opening the PR.

The events tests will go to `storage/src/events.rs`.